src/main.rs

```diff
@@ -225,7 +225,9 @@ impl Kopium {
                             if let Some(CustomResourceSubresources { status: Some(_), .. }) =
                                 version.subresources
                             {
-                                println!(r#"#[kube(status = "{}Status")]"#, kind);
+                                if self.has_status_resource(&structs) {
```

Comment:
might be able to combine these two if checks as:

`version.subresources.status.is_some() && self.has_status_resource(&structs)`

Reply:
I think I've done that in the latest commit. Please check if that is what you had in mind..

Reply:
ah, you need to chain the some-checks. perfect!

Reply:
ah, rustfmt is a bit annoying with nightly, try running `just fmt`

Reply:
ah yeah sorry ... I should not edit stuff in a web browser :D ... give me a second

Reply:
ok fixed the clippy/compile errors and ran `just fmt`

Reply:
Looks great. Hairier than i had originally suggested, but such is life with nested options :upside_down_face: 

Thanks a lot!

Reply:
yep yep can't do much about it - thanks for review & the quick feedback!